source/_patterns/02-molecules/interactive/read-more/read-more.twig

```diff
@@ -1,11 +1,19 @@
 {% if id and content %}
-  <div class="jcc-read-more">
+  <div class="jcc-read-more {% if inverted %} jcc-read-more--inverted{% endif %}">   
     <button class="jcc-read-more__trigger usa-button usa-button--unstyled" data-a11y-toggle="{{ id }}">
       {{ title ?? 'Read more'|t }}
+
+      {# Temp Icon #}
+      {# This will be replaced when the icons system is in place #}
+      <svg viewBox="0 0 24 24" class="icon">
+        <path d="M5.293 9.707l6 6c0.391 0.391 1.024 0.391 1.414 0l6-6c0.391-0.391 0.391-1.024 0-1.414s-1.024-0.391-1.414 0l-5.293 5.293-5.293-5.293c-0.391-0.391-1.024-0.391-1.414 0s-0.391 1.024 0 1.414z"></path>
+      </svg>
+      {# Temp Icon #}
```

Comment:
Totally fine with this being temporary, but curious why this isn't using [icon.twig](https://github.com/Exygy/courtyard/blob/master/source/_patterns/01-atoms/icons/icon.twig) now? 

Another option for dealing with the icon/text placement is to build that into the `icon.twig` component. Just an idea that can be handled in a follow up if you think that's a good idea.

Reply:
This will be updated with the final iconset soon. Just cutting some corners.